src/Extension/Mention/Mention.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the league/commonmark package.
+ *
+ * (c) Colin O'Dell <colinodell@gmail.com>
+ *
+ * Original code based on the CommonMark JS reference parser (https://bitly.com/commonmark-js)
+ *  - (c) John MacFarlane
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace League\CommonMark\Extension\Mention;
+
+use League\CommonMark\Inline\Element\Link;
+use League\CommonMark\Inline\Element\Text;
+
+class Mention extends Link
+{
+    /** @var string */
+    private $symbol;
+
+    /** @var string */
+    private $handle;
```

Comment:
I know I called them `handles` in my code, but I think a better term might be `identifier` because not everything will be a username/handle but rather some sort of unique identifier (which could be a handle or something else entirely, like a numeric ID).  What are your thoughts?

Reply:
How about `match` instead, since that is what is being returned from the regular expression: what it matches.